canonical-web-and-design/vanilla-framework
# Add suru strips to Vanilla

## Done

- Added new strip classes: p-strip--suru and p-strip--suru-topped
- Added new example templates for the new classes
- Updated the documentation to include the new classes

Fixes #2539 

## QA

- Pull code
- Run `./run`
- Open http://0.0.0.0:8101/ or [demo]
- Navigate to http://0.0.0.0:8101/docs/examples/patterns/strips/suru and ensure that the pattern is congruent with the [specification](https://github.com/canonical-web-and-design/vanilla-framework/issues/2539) and [meeting notes](https://github.com/canonical-web-and-design/meeting-notes/issues/596).
  - Do the same for http://0.0.0.0:8101/docs/examples/patterns/strips/suru-topped
- Navigate to the strip section in the [documentation](http://0.0.0.0:8101/docs/patterns/strip) and ensure that the copy is sensible and consistent in style to the rest of the documentation.
- Ensure that the code is sensible and consistent in style to the rest of the code.

## Screenshots

![image](https://user-images.githubusercontent.com/52562977/88785824-98a69480-d189-11ea-8d39-904b83c245b9.png)


![image](https://user-images.githubusercontent.com/52562977/88785872-ae1bbe80-d189-11ea-8acc-2454ece29bc6.png)
